internal/command/metrics/serve.go

```diff
@@ -63,7 +63,12 @@ func getAllMachineMetrics(ctx context.Context, aports *ports.Collection, query m
 	}
 
 	for _, machine := range machines {
-		metrics, err := aports.Provider.Metrics(ctx, machine.ID)
+		provider, ok := aports.MicrovmProviders[machine.Spec.Provider]
+		if !ok {
+			return nil, fmt.Errorf("microvm provider %s isn't available", machine.Spec.Provider)
```

Comment:
return here or just `continue` and log in case other machines are using a different, present, provider?

Reply:
Thats a good idea, done :+1: